Introduce kvstoremesh, a clustermesh-apiserver companion component (#26083)
cilium-team/cilium: Documentation/observability/metrics.rst

Cluster Mesh API Server Metrics
===============================

Cluster Mesh API Server metrics provide insights into the state of the

> ``clustermesh-apiserver`` process, the ``kvstoremesh`` process (if enabled)

Reviewer: Serial comma nit
```suggestion
``clustermesh-apiserver`` process, the ``kvstoremesh`` process (if enabled),
```

``kvstore_sync_queue_size``              ``scope``, ``source_cluster``                Number of elements queued for synchronization in the kvstore
``kvstore_initial_sync_completed``       ``scope``, ``source_cluster``, ``action``    Whether the initial synchronization from/to the kvstore has completed
======================================== ============================================ ========================================================

.. _kvstoremesh_metrics_reference:

kvstoremesh
-----------

Configuration
^^^^^^^^^^^^^

To expose any metrics, invoke ``kvstoremesh`` with the
``--prometheus-serve-addr`` option. This option takes a ``IP:Port`` pair but

> passing an empty IP (e.g. ``:9964``) will bind the server to all available

Reviewer: Use present tense
```suggestion
passing an empty IP (e.g. ``:9964``) binds the server to all available
```

> interfaces (there is usually only one in a container).

Reviewer: One what? Please verify.
```suggestion
interfaces (there is usually only one interface in a container).
```
